ember-cli-build.js

```diff
@@ -1,49 +1,26 @@
 'use strict';
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
-const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
-
-const isProductionBuild = process.env.EMBER_ENV === 'production';
 
 module.exports = function (defaults) {
-  let app = new EmberApp(defaults, {
+  const app = new EmberApp(defaults, {
+    'ember-cli-babel': { enableTypeScriptTransform: true },
     '@appuniversum/ember-appuniversum': {
       disableWormholeElement: true,
     },
+    // Add options here
   });
 
   const { Webpack } = require('@embroider/webpack');
-
-  const embroiderOptions = {
+  return require('@embroider/compat').compatBuild(app, Webpack, {
     staticAddonTestSupportTrees: true,
     staticAddonTrees: true,
-    staticHelpers: true,
-    staticModifiers: true,
-    staticComponents: true,
-    staticEmberSource: isProductionBuild,
-
-    // This config is needed to work around an Ember Data v4.12 + Embroider issue
-    // We can remove it once the fix is released: // https://github.com/embroider-build/embroider/issues/1506
-    compatAdapters: new Map([['@ember-data/debug', null]]),
-    packagerOptions: {
-      webpackConfig: {
-        plugins: [
-          process.env.ANALYZE_BUNDLE ? new BundleAnalyzerPlugin() : null,
```

Comment:
Whoops, one last thing. The bundle analyzer is no longer used now, so I would remove the dependency and `build:analyze-bundles` script in the package.json file.